src/views/Home/components/CLWCreditsDaysLeft/index.jsx

```diff
@@ -158,7 +158,13 @@ const CLWCreditsDaysLeft = () => {
       <CardContent>
         <Grid container direction="row" alignItems="center">
           <Grid item xs={7} align="left">
-            <CardHeader title={currSessionDescription} />
+            <CardHeader
+              title={currSessionDescription}
+              style={{
+                backgroundColor: gordonColors.primary.blue,
+                color: gordonColors.neutral.grayShades[50],
+              }}
```

Comment:
Please do not use inline styles. Instead, it would be better to add a CSS class for the header that can be re-used across the site.

You can add a class (such as "gc360_card_header") into `src/app.global.scss` and it will work no matter what component you use it from.